lib/components/Modal/index.tsx

```diff
@@ -104,11 +106,11 @@ class ModalImpl extends React.PureComponent<Props, {}> {
   render() {
     const {
       classes, title, description, classNameModalBody, onCancel, cancelContent, onContinue, continueContent,
-      isLoading, children, continueDanger,
+      isLoading, children, continueDanger, disableWrapperClick,
     } = this.props;
 
     return (
-      <ModalWrapper onCancel={onCancel}>
+      <ModalWrapper onCancel={(disableWrapperClick) ? undefined : onCancel}>
```

Comment:
@Nacryn Don't you mean `(disableWrapperClick) ? undefined : (onWrapperCancel || onCancel)` ?

Reply:
Ah indeed, I started adding `onWrapperCancel`at first but didn't push through in the end, so forgot to take it into account. But you have a point, I will update that 👍